bpaingest/projects/stemcells/ingest.py

```diff
@@ -239,20 +239,23 @@ def _get_resources(self):
 class StemcellsSingleCellRNASeqMetadata(BaseMetadata):
     contextual_classes = [StemcellsSingleCellRNASeq]
     metadata_urls = ['https://downloads-qcif.bioplatforms.com/bpa/stemcell/raw/single_cell_rnaseq/']
+    # //this is based on splitting of subdirs ( so cannot be used for anything else e.g., flow_id) as they appear in downloads
     metadata_url_components = ('facility_code', 'ticket')
```

Comment:
it'd be better to put this comment somewhere more universal, it doesn't apply only just to this class.

what's the `//` at the start of the comment?